src/test/java/gorest/get/GetUserTest.java

```diff
@@ -0,0 +1,29 @@
+package gorest.get;
+
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.Title;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.core.annotations.WithTags;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import services.gorest.actions.user.GetUser;
+
+@RunWith(SerenityRunner.class)
+@WithTags({
+        @WithTag(type = "service", name = "GoRest"),
+        @WithTag(type = "type", name = "Smoke"),
+        @WithTag(type = "type", name = "Regression"),
+        @WithTag(type = "type", name = "User")
+})
+public class GetUserTest {
+
+    @Steps
+    GetUser getUser;
+
+    @Title("Get a user with a known id")
+    @Test
+   public void getUserTest(){
+        getUser.getUserById(1799);
```

Comment:
Is this only for testing your code? Cause if it's a complete test case, this is wrong.